### DISCUSSION TITLE
2,000,000 sat channel closing, showing 0 local and 0 remote balance

### DISCUSSION THREAD

**Discussion by puck2** [lightningnetwork/lnd#7755] [Troubleshooting]
Opening transaction: f30f2e71833f7acd25b56edc073b94783785cb91e501128cae9c193ac99b9431

Channel open transaction address: bc1qf6mnra2d7p5fq9ncvg9wn4y2mq3wgxhftdyg7ud3a4p9dcc3xjjqrz4049

LNDg interface shows closing channel, but only references opening transaction in tooltip (which is very odd):

https://i.imgur.com/Z6l6f5B.png

Otherwise, shows ZERO local balance, ZERO remote balance, ZERO balance in limbo, and ZERO local commit fee

What's going on with this channel? It is one of the first ones I opened so I've lost track of whether the funds have actually returned to me or not.

**Comment by puck2** [comment]
Screenshot of channel closing in Thunderhub: https://i.imgur.com/5mIat8Z.png

But shouldn't there be a closing channel transaction?

**Reply by guggero** [reply]
There might have been an issue with the high fees recently that caused the closing transaction to disappear. You can try force closing the channel.

**Comment by puck2** [comment]
It was already forced closed when I reset my node

**Reply by guggero** [reply]
Did you already try restarting your node to force a re-publish of the force closing transaction? With the fees now coming down, perhaps it should work now. Please also check your log on startup to see if something is happening.

**Reply by puck2** [reply]
Yes, the problem is there is no closing transaction that I can see. Via Zeus I can see the two associated UTXOs of stuck on-chain funds.

**Reply by guggero** [reply]
Did you check your logs on restart whether `lnd` tried to re-broadcast a force closing transaction?

**Reply by puck2** [reply]
Here is the channel in question: https://1ml.com/channel/766369500249784321

Looking at logs, not sure whay I'm looking for, I see a lot like this:

`lnd_1         | 2023-07-01 20:31:06.192 [ERR] CRTR: process network updates got: unable to fetch utxo for chan_id=#############`

None of the logs reference the above mentioned channel in question.
